diff --git a/lib/tools.js b/lib/tools.js
index 05798cee5420b2d125fff347a9ebe4cfd4023fef..e3fa1055b686f4b35f7553d53a3ecf27632c2c0f 100644
--- a/lib/tools.js
+++ b/lib/tools.js
@@ -222,7 +222,7 @@ const tools = {
                 existing.path = folder.path;
                 existing.subscribed = !!folder.subscribed;
                 existing.listed = !!folder.listed;
-                existing.status = !!folder.status;
+                existing.status = folder.status;
 
                 if (folder.specialUse) {
                     existing.specialUse = folder.specialUse;
@@ -242,7 +242,7 @@ const tools = {
                     path: folder.path,
                     subscribed: !!folder.subscribed,
                     listed: !!folder.listed,
-                    status: !!folder.status
+                    status: folder.status
                 };
 
                 if (folder.delimiter) {
